src/commonTest/kotlin/fr/acinq/eclair/channel/states/ClosingTestsCommon.kt

```diff
@@ -1724,7 +1739,13 @@ class ClosingTestsCommon : EclairTestSuite() {
                 val (alice1, actions1) = alice.process(ChannelEvent.ExecuteCommand(CMD_FORCECLOSE))
                 assertTrue(alice1 is Closing)
                 assertEquals(7, actions1.size)
-                assertTrue(actions1.contains(ChannelAction.Storage.StoreOutgoingAmount(alice1.commitments.localCommit.spec.toLocal)))
+
+                val channelBalance = alice1.commitments.localCommit.spec.toLocal
+                if (channelBalance > 0.msat) {
+                    val ledgerPayment = actions1.filterIsInstance<ChannelAction.Storage.StoreChannelClosing>().firstOrNull()
```

Comment:
We don't use `ledger`, maybe `onChainPayment` is more consistent? Same in other tests where it's used.

Reply:
Done in b155d3d